docs/spec/manifest-v2-2.md

```diff
@@ -89,6 +89,11 @@ image manifest based on the Content-Type returned in the HTTP response.
             The os field specifies the operating system, for example
             `linux` or `windows`.
 
+        - **`osversion`** *string*
```

Comment:
Does os_version fit the bill?

Ultimately this is up to you all. I'm happy to choose os.version if that's what you'd like.